SQL/0000-00-00-schema.sql

```diff
@@ -688,32 +688,41 @@ CREATE TABLE `mri_protocol` (
   `Center_name` varchar(4) NOT NULL default '',
   `ScannerID` int(10) unsigned NOT NULL default '0',
   `Scan_type` int(10) unsigned NOT NULL default '0',
-  `TR_range` varchar(255) default NULL,
-  `TE_range` varchar(255) default NULL,
-  `TI_range` varchar(255) default NULL,
-  `slice_thickness_range` varchar(255) default NULL,
-  `FoV_x_range` varchar(255) default NULL,
-  `FoV_y_range` varchar(255) default NULL,
-  `FoV_z_range` varchar(255) default NULL,
-  `xspace_range` varchar(255) default NULL,
-  `yspace_range` varchar(255) default NULL,
-  `zspace_range` varchar(255) default NULL,
-  `xstep_range` varchar(255) default NULL,
-  `ystep_range` varchar(255) default NULL,
-  `zstep_range` varchar(255) default NULL,
-  `time_range` varchar(255) default NULL,
+  `TR_min` DECIMAL(10,4) DEFAULT NULL,
+	`TR_max` DECIMAL(10,4) DEFAULT NULL,
+	`TE_min` DECIMAL(10,4) DEFAULT NULL,
+	`TE_max` DECIMAL(10,4) DEFAULT NULL,
+	`TI_min` DECIMAL(10,4) DEFAULT NULL,
+	`TI_max` DECIMAL(10,4) DEFAULT NULL,
+	`slice_thickness_min` DECIMAL(9,4) DEFAULT NULL,
+	`slice_thickness_max` DECIMAL(9,4) DEFAULT NULL,
+	`xspace_min` int(4) DEFAULT NULL,
+	`xspace_max` int(4) DEFAULT NULL,
+	`yspace_min` int(4) DEFAULT NULL,
+	`yspace_max` int(4) DEFAULT NULL,
+	`zspace_min` int(4) DEFAULT NULL,
+	`zspace_max` int(4) DEFAULT NULL,
+	`xstep_min` DECIMAL(9,4) DEFAULT NULL,
+	`xstep_max` DECIMAL(9,4) DEFAULT NULL,
+  `ystep_min` DECIMAL(9,4) DEFAULT NULL,
+  `ystep_max` DECIMAL(9,4) DEFAULT NULL,
+  `zstep_min` DECIMAL(9,4) DEFAULT NULL,
+  `zstep_max` DECIMAL(9,4) DEFAULT NULL,
   `series_description_regex` varchar(255) default NULL,
+	`time_min` int(4) DEFAULT NULL,
```

Comment:
minor but please put series_description in the end (to maintain how it was originally)

Reply:
Hm, I did it that way because `time_range` followed `series_description_regex` in my original `mri_protocol` table. 
<img width="487" alt="screen shot 2018-03-26 at 11 52 57 pm" src="https://user-images.githubusercontent.com/17363738/37946002-03a42f62-3151-11e8-9047-afab0c67536e.png">

Reply:
I agree with Mouna, would be best to have series_description at the end of the table.